### PR TITLE
Add two_sided flag to get_clusters_table

### LINKED ISSUE #2547: nistats: adding two_sided flag to get_clusters_table would result in more consistent API?
Hello,

I've been recently using nistats to report results from fMRI data analysis. I wanted to perform two-sided test for my 2nd level spm, since I was interested in both increased and decreased brain responses. `map_threshold` function provide flag `two_sided` which nicely adjust significance level preserving clusters with both positive and negative test statistic values. 

Next I wanted to use `get_clusters_table` function to pool information about peak coordinates. However, this function can only report peaks above `stat_threshold` (basically neglecting negative test stat values). I made a workaround to make it consistent with `map_threshold` – I multiplied my spm by (-1), reapplied `get_clusters_table` and then merged two tables together. 

Do you think that adding `two_sided` flag and its functionality to `get_clusters_table` is a good idea?

### PR DESCRIPTION
**Pull request by tsalo**
Closes #2547.

Changes proposed in this pull request:

- Add a two_sided flag (default=True) to `nilearn.reporting._get_clusters_table()`.
- Update tests.
- Use two_sided thresholding in `glm_reporter`.

### CODE REVIEW

File: nilearn/reporting/_get_clusters_table.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def _get_val(row, input_arr):
 
 
 def get_clusters_table(stat_img, stat_threshold, cluster_threshold=None,
-                       min_distance=8.):
+                       two_sided=True, min_distance=8.):
```
Comment (bthirion) [inline nilearn/reporting/_get_clusters_table.py:134]: I think I'd default to False for the sake of bw compatibility.

**Review by bthirion — COMMENTED** [review]
Looks good otherwise, thx !

File: nilearn/reporting/_get_clusters_table.py (hunk 1 above)
Comment (tsalo) [reply nilearn/reporting/_get_clusters_table.py]: Good point.

**Review by NicolasGensollen — COMMENTED** [review]
The changes look good to me, thanks! :+1: 
I think we need to expose `two_sided` arg to `make_glm_report`, right?
Also, this probably needs a whats_new entry.

**Comment by tsalo** [comment]
Thanks to @bthirion and @NicolasGensollen for reviewing. I've added the What's New entry and `two_sided` is now a parameter for `make_glm_report` as well.

**Comment by NicolasGensollen** [comment]
I think you need to change the reference in `whats_new` to be `nilearn.reporting.make_glm_report` and `nilearn.reporting.get_clusters_table`. This should fix the broken CI.

**Review by bthirion — COMMENTED** [review]
Has CodeCov gone crazy ?
The PR LGTM otherwise.

**Review by NicolasGensollen — APPROVED** [review]
LGTM, thanks ! :+1:

**Pull request merged**